guis/desktop/gui.py

```diff
@@ -62,7 +63,7 @@ def flip(self):
 
 #################################
 # GUI
-class ProjpickerGui(wx.Frame):
+class ProjPickerGUI(wx.Frame):
```

Comment:
Good change - https://stackoverflow.com/questions/2853531/how-do-you-pep-8-name-a-class-whose-name-is-an-acronym